Ubuntu 22.04

tonyfruzza opened the pull request:
After finding additional changes were required to get #2 functional originally by @halotroop2288, I'm contributing back what I found worked for me. If this PR is accepted, do not merge #2.

Modern enough Ubuntu release to work with VSCode and github. This release required a couple additional considerations:

* Local time zone to be set to prevent being prompted during package installs at build time
* bzip2 package which is required by the vita sdk install script

This provides a recent enough version of glibc to be compatible with vscode while still maintaining compatibility with vita sdk

tonyfruzza committed:
Update to Ubuntu 22.04 and install necessary dependencies

tonyfruzza committed:
Fix Dockerfile formatting by ensuring a newline at the end of the file

halotroop2288 commented:
Could you add a `Co-authored by:` tag to the commit description if this is based on my work? Thanks for fixing it up.

tonyfruzza commented:
That makes sense to me, perhaps when it's squashed/merged that person doing so can. Unless you're suggesting I make additional commits? 

Based on the fact that this repo hasn't had an update in 6 years by @gnuton I'm not optimistic about anything being merged here 😿

tonyfruzza committed:
Update Dockerfile and README.md for Ubuntu 22.04 compatibility

tonyfruzza commented:
* Added back in `cmake` from kitware
* Updated [README.md](README.md)
* Resolved [Dockerfile](Dockerfile) build warnings about using older standards

tonyfruzza committed:
remove trailing white spaces

gnuton commented:
@tonyfruzza ar

> That makes sense to me, perhaps when it's squashed/merged that person doing so can. Unless you're suggesting I make additional commits?
> 
> Based on the fact that this repo hasn't had an update in 6 years by @gnuton I'm not optimistic about anything being merged here 😿

Merged! :D
